addons/swap-local-global/addon.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "Switch variables between \"For all sprites\" and \"For this sprite only\"",
+  "name": "Change variable scope",
```

Will users know what "scope" means?

Then again, there's always the description...

---

I think most people that requested this on feedback explained it fully, I'm not sure how many Scratchers know the name for it is "scope"

But I'm pretty sure that people that know what "scope" is will also understand the other name!

---

Long isn't necessarily bad.
Also, it's possible that some other addon name is longer in other languages.

---

> Switch variables between all sprites to one sprite only

What about 
```suggestion
  "name": "Switch variables between all sprites and this sprite",
```

At that point though it might be better to just keep the current name.